x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_form/follower_index_form.js

```diff
@@ -382,13 +389,13 @@ export const FollowerIndexForm = injectI18n(
           />),
           remoteClusterNotConnectedNotEditable: () => (<FormattedMessage
             id="xpack.crossClusterReplication.followerIndexForm.currentRemoteClusterNotConnectedCallOutDescription"
-            defaultMessage="You need to connect it before editing this follower index. Edit the remote cluster to
-              fix the problem."
+            defaultMessage="The remote cluster must be connected to edit this follower index."
```

Comment:
Can't edit follower index because remote cluster 'text' is not connected.

You can address this by editing the remote cluster.

Edit remote cluster